src/SprykerEco/Zed/Adyen/AdyenConfig.php

```diff
@@ -412,6 +412,23 @@ public function getMapperPaymentMethods(): array
         ];
     }
 
+    /**
+     * Specification:
+     * - Return Adyen api valid statuses for credit card payment status
```

Comment:
* Returns Adyen API valid statuses for credit card payment.

Reply:
Done